connectors/bigtable/flink-connector-gcp-bigtable/src/main/java/com/google/flink/connector/gcp/bigtable/serializers/GenericRecordToRowMutationSerializer.java

```diff
@@ -23,7 +23,9 @@
 
 import com.google.cloud.bigtable.data.v2.models.RowMutationEntry;
 import com.google.flink.connector.gcp.bigtable.utils.BigtableUtils;
+import com.google.flink.connector.gcp.bigtable.utils.ErrorMessages;
 import com.google.protobuf.ByteString;
+import io.grpc.Internal;
```

Comment:
i'm struggling to figure out what this is used for lol. also, do we need the internal version?

Reply:
THis should be `import org.apache.flink.annotation.Internal;`, the IDE messed things up as I was moving code from my other repo to this branch.

It doesn't really add anything, it's just an Annotation